services/ui-src/src/components/accordions/DueDateDropdown.tsx

```diff
@@ -45,29 +36,16 @@ export const DueDateDropdown = ({ templateName }: Props) => {
               </AccordionButton>
             </div>
             <AccordionPanel sx={sx.accordionPanel} className={mqClasses}>
-              <Flex sx={sx.accordionAdditionalTextFlex}>
-                <Text sx={sx.accordionDueDateText}>
-                  {templateName} Due Dates
-                </Text>
-                <Text sx={sx.accordionDescriptionText}>
-                  {dropDownTextValues.accordionDescriptionText}
-                </Text>
-                {dropDownTextValues.hasTable ? (
-                  <DueDateTable templateName={templateName} />
-                ) : (
-                  ""
-                )}
-                {dropDownTextValues.hasList ? (
+              <Flex sx={sx.accordionPanelFlex}>
+                <Text sx={sx.accordionBodyTitle}>{verbiage.bodyTitle}</Text>
+                <Text sx={sx.accordionBodyText}>{verbiage.bodyText}</Text>
+                {verbiage.table && <DueDateTable verbiage={verbiage.table} />}
+                {verbiage.list && (
```

Comment:
Testing for the existence of `table` and `list` here instead of having separate vars `hasTable` and `hasList`.